### PR TITLE
Make plot_history() function in episode 3 and 4 functional

### LINKED ISSUE #370: Make plot_history() functional by passing it History object
Currently the `plot_history()` function in episode 3 and 4 uses the `history` object that is defined somewhere in the notebook. It is much more explicit (and prevents mistakes by learners) to actually pass the history as an argument to the function.

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #370

### CODE REVIEW

File: episodes/3-monitor-the-model.Rmd [hunk 1]
```diff
@@ -359,19 +359,20 @@ We will create a function for it, because we will make use of this more often in
...
 
     Args:
-        metrics(str, list): Metric or a list of metrics to plot
+        history (keras History object that is returned by model.fit())
+        metrics (str, list): Metric or a list of metrics to plot
     """
     history_df = pd.DataFrame.from_dict(history.history)
     sns.lineplot(data=history_df[metrics])
     plt.xlabel("epochs")
     plt.ylabel("RMSE")
```
Comment (cpranav93) [inline episodes/3-monitor-the-model.Rmd:373]:
```suggestion
    plt.ylabel("metric")
```

We may not always want to use RMSE as the metric. Better to be more generic?

File: episodes/4-advanced-layer-types.Rmd [hunk 2]
```diff
@@ -372,23 +372,24 @@ We can plot the training process using the history:
...
     """
     Plot the training history
 
     Args:
+        history (keras History object that is returned by model.fit())
         metrics(str, list): Metric or a list of metrics to plot
     """
     history_df = pd.DataFrame.from_dict(history.history)
     sns.lineplot(data=history_df[metrics])
     plt.xlabel("epochs")
```
Comment (cpranav93) [inline episodes/4-advanced-layer-types.Rmd:386]:
```suggestion
    plt.ylabel("metric")
```

ylabel is missing. Adding similar to episode 3 to keep it consistent

**Review by cpranav93 — CHANGES_REQUESTED** [review]
A couple of suggestions. Your thoughts @svenvanderburg ?

File: episodes/3-monitor-the-model.Rmd (hunk 1 above)
Comment (svenvanderburg) [reply episodes/3-monitor-the-model.Rmd:373]: Yes, in this specific notebook it is always RMSE, but you are right that it is nicer to make it more generic.

**Comment by svenvanderburg** [comment]
Thanks for your comments @cpranav93 !
